src/DssSpell.sol

```diff
@@ -19,43 +19,45 @@ pragma solidity 0.8.16;
 import "dss-exec-lib/DssExec.sol";
 import "dss-exec-lib/DssAction.sol";
 
-interface RwaLiquidationOracleLike {
-    function tell(bytes32 ilk) external;
+interface VatLike {
+    function deny(address usr) external;
 }
 
-interface ChainlogLike {
-    function removeAddress(bytes32) external;
+interface CatLike {
+    function deny(address usr) external;
```

Comment:
Those interfaces are the same.
You could probably have one called `DenyLike` instead.
Not mandatory.

Reply:
adjusted